src/main/java/racingcar/RacingGame.java

```diff
@@ -0,0 +1,85 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+public class RacingGame {
+
+    // 각 Car들의 시작 위치
+    public final static int BASE_POSITION = 1;
```

Comment:
```suggestion
    public static final int BASE_POSITION = 1;
```

static final
http://cr.openjdk.java.net/~alundblad/styleguide/index-v6.html#toc-modifiers